From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Wed, 22 May 2019 15:25:06 -0400
Subject: Drop sq/unsq man pages

Avoid shipping manual pages for sq and unsq, since we are no longer shipping them
---
 Makefile | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/Makefile b/Makefile
index 0d0fcb3..a1195d8 100644
--- a/Makefile
+++ b/Makefile
@@ -400,13 +400,10 @@ install-dictbuild:
 	    rm -f fixispell-a$$MAN1EXT
 	@. ./config.sh; \
 	  set -x; \
-	  $$INSTALL sq.1 $(DESTDIR)$$MAN1DIR/sq$$MAN1EXT; \
 	  for m in buildhash munchlist findaffix tryaffix; do \
 	    echo ".so `basename $$MAN1DIR`/ispell$$MAN1EXT" \
 	      > $(DESTDIR)$$MAN1DIR/$$m$$MAN1EXT; \
-	  done; \
-	  echo ".so `basename $$MAN1DIR`/sq$$MAN1EXT" \
-	    > $(DESTDIR)$$MAN1DIR/unsq$$MAN1EXT 
+	  done
 	@. ./config.sh; \
 	  set -x; \
 	  cd $(DESTDIR)$$MAN1DIR; \
